### ISSUE TITLE
Land use: night-time economy maps

### ISSUE THREAD

**Issue by MKIM1008** [colouring-cities/colouring-britain#350]
TBC 

Nightime shops - security connection

i.e. we could map shops:
a. placed for 24 hours or late
b. open very early 
c, normal open

**Comment by MKIM1008** [comment]
Type of economy, SME (business, marketing, urban study, etc.), CCTV

**Comment by MKIM1008** [comment]
@matkoniecz I will see OSM first and how to derive the operation time. If the data is collected via Google Maps exploring the town centre online, is it okay to map the data on C.L? (license)

**Comment by matkoniecz** [comment]
> Also, if the data is collected via Google Maps exploring the town centre online, is it okay to map the data on C.L? (license)

It definitely violates Google's terms of use and they do not give permission to do this.

Distributing it on ODBL license like CL does would be quite dubious.

**Comment by matkoniecz** [comment]
If you ask about looking at Google Street View: it definitely violates Google's terms of use and they do not give permission to do this.

But here it is more dubious whether Google has right to do this. But I ma not looking for legal dispute with Google.

warning: I am not a lawyer.

**Comment by polly64** [comment]
Yes agree we shouldn’t be accessing data from any commercial unless they have specifically released under open licences

**Comment by MKIM1008** [comment]
![image](https://github.com/user-attachments/assets/58a649db-0e03-4720-93a3-7cccb3d5be4a)
 
136 business opening times in Loughborough are collected from OpenStreetMap (1/6)

Business types: 
![image](https://github.com/user-attachments/assets/37d0ab25-0d1f-42c7-b0c9-5d2af9c6d3cf)
![image](https://github.com/user-attachments/assets/584501d6-976d-4bc3-bcc4-f090ed603255)
![image](https://github.com/user-attachments/assets/b9be773f-7075-46a4-8651-03dc754005c9)

**Comment by MKIM1008** [comment]
London example:
![image](https://github.com/user-attachments/assets/cd50c6c5-f5fd-4c5a-8ea7-bcdf66769b8e)
19088 business opening times in Greater London are collected from OpenStreetMap (- allowed to collect data limited)
![image](https://github.com/user-attachments/assets/ca575160-1b63-4487-b085-947b1f84c843)

**Comment by MKIM1008** [comment]
Useful links from Mateusz: 
https://overpass-turbo.eu/s/1PCp
https://overpass-turbo.eu/s/1PCs
https://wiki.openstreetmap.org/wiki/Key:opening_hours
https://openingh.openstreetmap.de/evaluation_tool/?setLng=en
https://overpass-turbo.eu/s/1PCx
https://wiki.openstreetmap.org/wiki/Key:shop
https://wiki.openstreetmap.org/wiki/Key:amenity
